watchdog/src/config.rs

```diff
@@ -101,8 +83,8 @@ impl Config {
     pub fn testnet() -> Self {
         Self {
             bitcoin_network: BitcoinNetwork::Testnet,
-            blocks_behind_threshold: BLOCKS_BEHIND_THRESHOLD,
-            blocks_ahead_threshold: BLOCKS_AHEAD_THRESHOLD,
+            blocks_behind_threshold: 100,
+            blocks_ahead_threshold: 100,
             min_explorers: MIN_EXPLORERS,
             bitcoin_canister_principal: Principal::from_text(TESTNET_BITCOIN_CANISTER_PRINCIPAL)
                 .unwrap(),
```

Comment:
We should remove `BitcoinBlockApi::ApiBitapsComTestnet` from testnet explorers since it was misbehaving for 30+ days and stopped providing data recently at all.

![image](https://github.com/dfinity/bitcoin-canister/assets/103510076/4b79a0f9-b5dc-4b30-83da-406b060bb5aa)

Reply:
Sure, maybe then we reduce the min explorers for testnet from 3 to 2?

Reply:
SGTM reducing to 2 for the testnet.

Reply:
Done.